apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactNewEdit.tsx

```diff
@@ -119,9 +120,12 @@ const ContactNewEdit = ({ id, data, close }: ContactNewEditProps): ReactElement
 	};
 
 	const isPhoneValid = async (phone: string): Promise<boolean | string> => {
-		if (!phone || initialValue.phone === phone) {
+		if (phone === initialValue.phone) {
```

Comment:
this changes the logic, I'm not confident if this is expected elsewhere or not.

previously an empty input would've returned true, despite of if it matches initialValue or not. e.g. `isPhoneValid('')` would've been true even if `initialValue.phone != ''` which isn't the case anymore here.

cc @RocketChat/omnichannel

Reply:
ok